dipy/tracking/learning.py

```diff
@@ -2,113 +2,113 @@
 import numpy as np
 import dipy.tracking.distances as pf
 
-def detect_corresponding_tracks(indices,tracks1,tracks2):
+
+def detect_corresponding_tracks(indices, tracks1, tracks2):
     ''' Detect corresponding tracks from list tracks1 to list tracks2
     where tracks1 & tracks2 are lists of tracks
-    
+
     Parameters
     ------------
     indices : sequence
        of indices of tracks1 that are to be detected in tracks2
-    tracks1 : sequence 
+    tracks1 : sequence
        of tracks as arrays, shape (N1,3) .. (Nm,3)
-    tracks2 : sequence 
+    tracks2 : sequence
        of tracks as arrays, shape (M1,3) .. (Mm,3)
-            
+
     Returns
     ---------
-    track2track : array (N,2) where N is len(indices) of int 
-       it shows the correspondance in the following way:       
+    track2track : array (N,2) where N is len(indices) of int
+       it shows the correspondance in the following way:
        the first column is the current index in tracks1
        the second column is the corresponding index in tracks2
-    
+
     Examples
     ----------
     >>> import numpy as np
     >>> import dipy.tracking.learning as tl
     >>> A=np.array([[0,0,0],[1,1,1],[2,2,2]])
     >>> B=np.array([[1,0,0],[2,0,0],[3,0,0]])
-    >>> C=np.array([[0,0,-1],[0,0,-2],[0,0,-3]])    
+    >>> C=np.array([[0,0,-1],[0,0,-2],[0,0,-3]])
```

Comment:
Whitespace still needed in these examples (e.g. around equal signs, after commas)